src/vl-map-wms-layer.js

```diff
@@ -10,6 +10,7 @@ import {VlMapLayer} from '/src/vl-map-layer.js';
  * @property {string} data-vl-url - Attribuut bepaalt de WMS url. Verplicht.
  * @property {string} data-vl-layers - Attribuut bepaalt de layers van de WMS. Verplicht.
  * @property {string} [data-vl-styles=] - Attribuut bepaalt de WMS stijlen.
+ * @property {string} [data-vl-sld-body=] - Attribuut bepaalt de SLD stijl.
```

Comment:
SLD hier misschien best voluit schrijven en evt. nog wat uitleg schrijven waarom ge een sld-body zou willen setten.

Reply:
wat is een sld body

Reply:
Ik heb het verduidelijkt